### PR TITLE
Launchpad: Add the Navigator package structure

### PR DESCRIPTION
**Pull request by valterlorran**
Related to #

## Proposed Changes

* This PR adds the basic structure for publishing the Launchpad Navigator package. As part of our project, we need to publish this package so we can easily use it on jetpack-mu-wpcom.
* This will be made possible by Dale's PR: https://github.com/Automattic/jetpack/pull/33045

More info:
https://github.com/Automattic/wp-calypso/blob/40811b2929612ebb529ee4bff7ce51bdc61c51e7/docs/monorepo.md#publishing

## Testing Instructions

* Visual inspection of the changed files

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: packages/launchpad-navigator/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+```js
+function renderApp() {
+    const [ showNavigator, setShowNavigator ] = useState( false );
+    return (
+        <>
+            <Button onClick={ setShowNavigator( ! showNavigator ) }>
+                <LaunchpadNavigatorIcon siteSlug={ siteSlug } />
+            </Button>
+            { showNavigator && (
+                <FloatingNavigator siteSlug={ siteSlug } />
```
Comment (daledupreez) [inline packages/launchpad-navigator/README.md:29]: Nit: We're importing `LaunchpadNavigator` above, not `FloatingNavigator`.

**Review by daledupreez — COMMENTED** [review]
I also see that the `private` top-level field is set to `true` - can you check if that's what we're doing for other packages that we publish to NPM?

**Comment by valterlorran** [comment]
@daledupreez thanks for catching the `private` key. We shouldn't have it set true.

File: packages/launchpad-navigator/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Launchpad Navigator
+
+This package provides the components for attaching the Navigator of the 
+Launchpads to the master bar.
```
Comment (daledupreez) [inline packages/launchpad-navigator/README.md]: I don't believe this is strictly true. This package provides the components for showing and interacting with one or more Launchpad task lists for a given site.

**Review by daledupreez — APPROVED** [review]
LGTM!

**Pull request merged**